model_analyzer/result/model_config_measurement.py

```diff
@@ -102,6 +102,13 @@ def model_config_name(self):
 
         return self._model_config_name
 
+    def model_name(self):
+        """
+        Return the model name
+        """
+
+        return self._model_config_name.partition("_config_")[0]
```

Comment:
Can you add a comment that this should be replaced with the extract_model_name_from_variant_name() static method, once the ensemble code gets merged to main

Reply:
Added the following comment in method description,
```
TODO: This method should be replaced with the extract_model_name_from_variant_name() static method,
        once the ensemble code gets merged to main
```